install.sh

```diff
@@ -190,8 +190,7 @@ fi
 
 if ! egrep -q '^SQLALCHEMY_DATABASE_URI=' .env; then
     echo 'Setting SQLALCHEMY_DATABASE_URI'
-    # It's assumed DB_NAME, DB_USER, DB_PASS have been already captured above
-    echo "SQLALCHEMY_DATABASE_URI=mysql+pymysql://$DB_USER:$DB_PASS@localhost/$DB_NAME" >> .env
+    echo "SQLALCHEMY_DATABASE_URI=sqlite:///hushline.db" >> .env
```

Comment:
This should really have an absolute path like `/var/lib/hushline/sqlite.db` so it's not sitting in the app's code directory

Reply:
Added absolute path:
`echo "SQLALCHEMY_DATABASE_URI=sqlite:////var/lib/hushline/hushline.db" >> .env`

Added DB folder path and permissions:
```
# SQLite database directory and file path
SQLITE_DIR="/var/lib/hushline"
SQLITE_DB_FILE="$SQLITE_DIR/hushline.db"

# Ensure SQLite directory exists
if [ ! -d "$SQLITE_DIR" ]; then
    echo "Creating SQLite directory at $SQLITE_DIR..."
    mkdir -p "$SQLITE_DIR"
    # Ensure the directory is owned by the application user
    chown $HUSHLINE_USER:$HUSHLINE_GROUP "$SQLITE_DIR"
    echo "✅ SQLite directory created."
else
    echo "👍 SQLite directory already exists."
fi
```